### PR TITLE
Features/add helpers

### LINKED ISSUE #5: Add helper to retreive text direction
Some language need to be read from right to left (Arab, Hebr, etc.). So the package should offer a helper to retreive ltl or rtl.

### PR DESCRIPTION
**Pull request by Gandhi11**
Wait for #7 to be merged

Fix #5 and #6

### CODE REVIEW

File: src/LocaleService.php [hunk 1]
```diff
@@ -194,4 +194,128 @@ public function setSystemLocale(string $locale = null)
...
+     * @throws \Exolnet\Translation\TranslationException
+     */
+    public function getLocaleNativeName(string $locale = null)
+    {
+        if (!$locale) {
+            $locale = $this->getCurrentLocale();
+        }
+
+        if (!isset($this->getLocalesAvailable()[$locale])) {
+            return '';
```
Comment (martinblanchette) [inline src/LocaleService.php]: Is it a better idea to fallback on the `name` using `getLocaleName` method?

File: src/Blade/LanguageBladeExtension.php [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+    }
+
+    public function isRtl()
+    {
+        return !$this->isLtr();
+    }
+
+    public function isLtr()
+    {
+        return LaravelTranslation::getCurrentLocaleDirection() === "rtl";
```
Comment (martinblanchette) [inline src/Blade/LanguageBladeExtension.php]: Use single quote notation.

File: src/TranslationServiceProvider.php [hunk 3]
```diff
@@ -127,4 +136,33 @@ protected function getConfigFile(): string
...
+     */
+    protected function getViewsPath(): string
+    {
+        return __DIR__ .
+            DIRECTORY_SEPARATOR . '..' .
+            DIRECTORY_SEPARATOR . 'resources' .
+            DIRECTORY_SEPARATOR . 'views';
+    }
+
+    private function bootBladeExtensions()
```
Comment (martinblanchette) [inline src/TranslationServiceProvider.php]: 🙏 Separate nested `foreach` in order to make the code readable and much clean.

File: src/Blade/LanguageBladeExtension.php (hunk 2 above)
Comment (Gandhi11) [reply src/Blade/LanguageBladeExtension.php]: Should be fixed

File: src/TranslationServiceProvider.php (hunk 3 above)
Comment (Gandhi11) [reply src/TranslationServiceProvider.php]: Should be fixed

File: src/LocaleService.php (hunk 1 above)
Comment (Gandhi11) [reply src/LocaleService.php]: Should be fixed

**Pull request closed without merging**